### PR TITLE
ApiResponse implementation

### PR DESCRIPTION
**Pull request by lukasz-falda**
AC
---
* ApiResponse has full structure
* README was updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,37 +91,68 @@ $apiResponse = $request->changeAvailability(new ContentAvailability('product/url
 
 ---
 In all request you can catch some exceptions:
-* `ToManyItemsException` - make request with less items,
+* `BadRequestException` - bad request,
+* `ToManyItemsException` - make request with fewer items,
```
Comment (malarzm) [inline README.md]: `TooMany` - is this typo just here or in the exception as well?

File: README.md [hunk 2]
```diff
@@ -91,37 +91,68 @@ $apiResponse = $request->changeAvailability(new ContentAvailability('product/url
...
 }
 
 ```
 
 Response
 ------------
 
-//TODO - write it
+`\Answear\LuigisBoxBundle\Response\ApiResponse`:
+* (bool) `$success` - `true` if all documents will pass successfully,
```
Comment (malarzm) [inline README.md]:
```suggestion
* (bool) `$success` - `true` if all documents will be passed successfully,
```

File: README.md [hunk 3]
```diff
@@ -91,37 +91,68 @@ $apiResponse = $request->changeAvailability(new ContentAvailability('product/url
...
+`ApiResponseError`:
+* (string) `$url` - url of document
+* (string) `$type` - type of error (ex. `malformed_input`)
+* (string) `$reason` - failure text (ex. `incorrect object format`)
+* (array|null) `$causedBy` - specific reason of error (ex. `["url": ["is missing"]]`)
+
+
+Note!
+
+If some document will fail, `ApiResponse::$success` will be `false`. Check `$okCount` if you want to know how many documents were been updated and `$errors` to check exactly which documents fails.
```
Comment (malarzm) [inline README.md]:
```suggestion
`ApiResponse::$success` will be set to `false` if any of passed documents fails. Check `$okCount` if you want to know how many documents were updated and `$errors` to check exactly which documents failed.
```

File: README.md (hunk 1 above)
Comment (lukasz-falda) [reply README.md]: Everywhere :+1:

**Review by malarzm — APPROVED** [review]

**Review by bmulawa — APPROVED** [review]

**Review by jrawska — APPROVED** [review]

**Pull request merged**